production/src/lib.rs

```diff
@@ -20,17 +20,35 @@ struct ProdMigrate;
 #[async_trait]
 impl Migrate for ProdMigrate {
     async fn migrate(_pool: &PgPool) {
-        // sqlx::migrate!("./migrations").run(pool).await.unwrap()
-        todo!()
     }
 }
 
 struct ProdCourseSync;
 
 #[async_trait]
 impl CourseSync for ProdCourseSync {
-    async fn init_sync(_pool: PgPool) {
-        todo!()
+    async fn init_sync(pool: PgPool) {
```

Comment:
[understanding] What does this function do?

Reply:
Populates the DB when we first boot the app iirc.